monkey/monkey_island/cc/setup.py

```diff
@@ -0,0 +1,42 @@
+import logging
+
+from monkey_island.cc.services.attack.mitre_api_interface import MitreApiInterface
+from monkey_island.cc.models.attack.attack_mitigations import AttackMitigations
+from monkey_island.cc.database import mongo
+from pymongo import errors
+
+
+logger = logging.getLogger(__name__)
+
+
+def setup():
+    logger.info("Setting up the Monkey Island, this might take a while...")
+    try_store_mitigations_on_mongo()
+
+
+def try_store_mitigations_on_mongo():
+    mitigation_collection_name = AttackMitigations.COLLECTION_NAME
+    try:
+        mongo.db.validate_collection(mitigation_collection_name)
+        if mongo.db.attack_mitigations.count() == 0:
+            raise errors.OperationFailure("Mitigation collection empty. Try dropping the collection and running again")
+    except errors.OperationFailure:
+        try:
+            mongo.db.create_collection(mitigation_collection_name)
+        except errors.CollectionInvalid:
+            pass
```

Comment:
From the [docs](https://api.mongodb.com/python/current/api/pymongo/errors.html#pymongo.errors.CollectionInvalid) it seems like validation failed
Why aren't we raising on this, or at least logging?